lib/tarantool_server.py

```diff
@@ -551,6 +557,9 @@ def copy_files(self):
         shutil.copy('.tarantoolctl', self.vardir)
         shutil.copy(os.path.join(self.TEST_RUN_DIR, 'test_run.lua'),
                     self.vardir)
+        if self.ini.get('pre_cleanup', False):
```

Comment:
Can be simplified to self.ini['pre_cleanup']?

Reply:
Need to use get here because of nondefault servers doesn't have ini

Reply:
It seems that self.ini values used in *Server are mostly propagated to *Server fields, but I don't want to push you to rewrite it so.